pkg/admissioncontroller/webhooks/admission/seedrestriction/admission.go

```diff
@@ -220,6 +227,21 @@ func (h *handler) admitCertificateSigningRequest(seedName string, request admiss
 	return h.admit(seedName, &seedNameInCSR)
 }
 
+func (h *handler) admitClusterRoleBinding(ctx context.Context, seedName string, request admission.Request) admission.Response {
+	if request.Operation != admissionv1.Create {
+		return admission.Errored(http.StatusBadRequest, fmt.Errorf("unexpected operation: %q", request.Operation))
+	}
+
+	// Allow gardenlet to create cluster role bindings referencing service accounts which can be used to bootstrap other
+	// gardenlets deployed as part of the ManagedSeed reconciliation.
+	if strings.HasPrefix(request.Name, bootstraputil.ClusterRoleBindingNamePrefix) {
```

Comment:
Similar to my comment about the bootstrap token: What do you think about checking that the `RoleRef` is set to `GardenerSeedBootstrapper`?

Reply:
Valid point, thanks, done with 6418824